test/fiddle/test_function.rb

```diff
@@ -126,14 +126,16 @@ def call one
     end
 
     def test_last_error
-      if ffi_backend?
-        omit("Fiddle.last_error doesn't work with FFI backend")
-      end
+      require 'rbconfig/sizeof'
+
+      strtol = Function.new(@libc['strtol'], [TYPE_VOIDP, TYPE_VOIDP, TYPE_INT], TYPE_LONG)
+
+      assert_nil Fiddle.last_error unless ffi_backend?
```

Comment:
FWIW this only works on CRuby due to the `Fiddle.last_error = nil` in `setup`.
On the FFI backend there is no clean way to do this, and there seems to be little value to return `nil` on `Fiddle.last_error`, as that can only happen if:
* there never was a call using Fiddle yet
* the user called `Fiddle.last_error = nil` explicitly, which seems not really useful

Maybe setup should `Fiddle.last_error = 0` instead and then we could test that here? Let me know what you prefer.

Reply:
Let's use `Fiddle.last_error = 0`.

Reply:
Done